.github/workflows/frontend.yml

```
@@ -62,3 +62,27 @@ jobs:
       - name: Frontend Tests
         run: npm run ci:frontend:test
         working-directory: simulator-ui
+
+  test:
```

```suggestion
  e2e:
```

I would suggest another name 😉